content/applications/finance/accounting/get_started/chart_of_accounts.rst

```diff
@@ -104,61 +93,58 @@ right type among the following list:
 Assets, deferred expenses, and deferred revenues automation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Some **account types** display a new field **to automate** the creation of :ref:`assets
-<assets-automation>` entries, :ref:`deferred expenses <deferred-expenses-automation>` entries,
-and :ref:`deferred revenues <deferred-revenues-automation>` entries.
+Some **account types** can **automate** the creation of :ref:`assets <assets-automation>` entries,
+:ref:`deferred expenses <deferred-expenses-automation>` entries, and
+:ref:`deferred revenues <deferred-revenues-automation>` entries. To **automate** entries, click on
+:guilabel:`Setup` on an account line and go to the :guilabel:`Automation` tab.
 
-You have three choices for the :guilabel:`Automation` field:
+You have three choices for the :guilabel:`Automation` tab:
 
 #. :guilabel:`No`: this is the default value. Nothing happens.
 #. :guilabel:`Create in draft`: whenever a transaction is posted on the account, a draft entry is
    created, but not validated. You must first fill out the corresponding form.
-#. :guilabel:`Create and validate`: you must also select a **model**. Whenever a transaction is
-   posted on the account, an entry is created and immediately validated.
+#. :guilabel:`Create and validate`: you must also select a :guilabel:`Deferred Expense Model`.
+   Whenever a transaction is posted on the account, an entry is created and immediately validated.
 
 Default taxes
```

Comment:
For idiots like me, please specify where to go for this (click 'Setup')

Reply:
It's specified on line 99